homework-submission/week1/src/App.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import './App.css';
+import ListItem from './components/items';
+import ListHeader from './components/listHeader';
+import List from './components/list';
+const items = [
+  {
+    id: '76385kjsng2358',
+    description: 'Get out of bed',
+    deadline: '2017-09-11',
+    done: true,
+  },
+  {
+    id: '76385kjsng23453518',
+    description: 'Brush teeth',
+    deadline: '2017-09-10',
+    done: false,
+  },
+  {
+    id: '76385kjsng2397528',
+    description: 'Eat breakfast',
+    deadline: '2017-09-09',
+    done: false,
+  },
+];
+// I created List, ListHeader and ListItem Components.
+
+// I have  props : listName, description,deadline and done for both Static and Dynamic List.
```

Comment:
Better to move these comments, the documentation of your components to the code of your components itself. So that other developers reading you code get a better understanding of how that specific part works :-)